src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -92,9 +92,10 @@ const enum VSCodeOscPt {
 	 * with parsing out the command, such as conpty not guaranteeing the position of the sequence or
 	 * the shell not guaranteeing that the entire command is even visible.
 	 *
-	 * The command line should escape ascii characters using the `\xAB` format, where AB are the
+	 * The command line can escape ascii characters using the `\xAB` format, where AB are the
 	 * hexadecimal representation of the character code (case insensitive), and escape the `\`
-	 * character using `\\`. This is particularly important for new line and semi-colon.
+	 * character using `\\`. It's recommended to escape semi-colon (`0x3b`) and charcaters 0x20 and
```

Comment:
```suggestion
	 * character using `\\`. It's required to escape semi-colon (`0x3b`) and characters 0x20 and
```
"required" may be a bit too strongly worded, but not escaping newlines and semi-colons basically guarantees this feature to not work correctly.

Reply:
Required seems better, for semi-colons it probably will work in VS Code but is recommended, for new lines they will get translated into spaces I believe.